src/configuration-requestor.ts

```diff
@@ -41,22 +47,57 @@ export default class ConfigurationRequestor {
         createdAt: configResponse.createdAt,
       });
 
-      // TODO: different polling intervals for bandit parameters
-      const banditResponse = await this.httpClient.getBanditParameters();
-      if (banditResponse?.bandits) {
-        if (!this.banditModelConfigurationStore) {
-          throw new Error('Bandit parameters fetched but no bandit configuration store provided');
-        }
+      if (this.requiresBanditModelConfigurationStoreUpdate(configResponse.banditReferences)) {
```

Comment:
Can you check `banditReferences` at line 30/36
`const flagsHaveBandits = Object.keys(configResponse.banditReferences ?? {}).length > 0;`

`bandits` is deprecated and replaced with `banditReferences`. If we're able to 🪓 the use of `bandits` in this PR, that'd be great, but it might be too much noise.